DEV-Simple/views/ViewController.swift

```diff
@@ -75,7 +79,8 @@ class ViewController: UIViewController {
         return banner
     }()
 
-    var devToURL = "https://dev.to"
+    var devToURL = "http://localhost:3000"
```

Comment:
Can we make this an environment variable?

Reply:
Somewhat tricky (compared to how easy it is in Rails), but yes we can. I'll set the production URL as default and the environment variable would override it in `viewDidLoad()`. The ENV variable shouldn't have any effect in Production/TestFlight because it only works when deployed using XCode. It would just need to be manually set in the project scheme like this:

![Screen Shot 2020-02-17 at 11 52 38](https://user-images.githubusercontent.com/6045239/74676844-88bb6b00-517c-11ea-8671-14c494213df2.png)

Let me know if you have a better ENV Variable name to use in this case 😉